paymentsheet/src/test/java/com/stripe/android/model/AddressFixtures.kt

```diff
@@ -19,4 +24,67 @@ object AddressFixtures {
             """.trimIndent()
         )
     )
+
+    val ADDRESS_SCHEMA_ELEMENTS = AddressSchemas(
```

Comment:
Is it possible to use a more realistic value, rather than a small set of hardcoded values?